src/main/java/racingcar/model/Name.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.model;
+
+public class Name {
+
+    private static final String CAR_NAME_LONGER_THAN_FIVE = "[ERROR] 자동차 이름은 5글자를 초과할 수 없습니다.";
+    private static final String CAR_NAME_LESS_THAN_ONE = "[ERROR] 자동차 이름은 최소 1글자 이상이어야 합니다. ";
+    private static final int MINIMUM_CAR_NAME_LENGTH = 1;
+    private static final int MAXIMUM_CAR_NAME_LENGTH = 5;
+
+    private final String name;
+
+    public Name(String name) {
+        this.name = validateCarNameLength(name);
+    }
+
+    private String validateCarNameLength(String name) {
+        if (name.length() < MINIMUM_CAR_NAME_LENGTH) {
+            throw new IllegalArgumentException(CAR_NAME_LESS_THAN_ONE);
+        }
+        if (name.length() > MAXIMUM_CAR_NAME_LENGTH) {
+            throw new IllegalArgumentException(CAR_NAME_LONGER_THAN_FIVE);
+        }
+
+        return name;
+    }
```

Comment:
`validateXXX` 함수는 유효성 검증만 진행하도록하고 값의 초기화는 생성자에서 직접하도록 하는 것이 좋을 것 같아요. 🤗 (하나의 메서드가 하나의 기능만 담당하도록! 😉) 

```suggestion
    public Name(String name) {
        validateCarNameLength(name);
        this.name = name;
    }

    private void validateCarNameLength(String name) {
        if (name.length() < MINIMUM_CAR_NAME_LENGTH) {
            throw new IllegalArgumentException(CAR_NAME_LESS_THAN_ONE);
        }
        if (name.length() > MAXIMUM_CAR_NAME_LENGTH) {
            throw new IllegalArgumentException(CAR_NAME_LONGER_THAN_FIVE);
        }
    }
```

Reply:
넵!! 유효성 검증 메소드는 해당 역할만 하도록 변경했습니다!!